### PR TITLE
DisplayCal is starting on Windows10 with Python3.8

### PR DESCRIPTION
**Pull request by p5k369**
- Downloading and installing of argyll is working
- Displays and colorimeter are found
- Constant name "domain" changed to uppercase according to PEP8 and mostly because it is very confusing
- also used f-strings in modified code wherever possible
- #32 according to python3 documenation xmlparser object no longer has a 'returns_unicode' attribute: https://docs.python.org/3.9/library/pyexpat.html#xml.parsers.expat.ParserCreate
- Dict keys of win32api.GetMonitorInfo return are no bytestrings

- profiling and calibrating isn't working yet on win10

### CODE REVIEW

**Comment by eoyilmaz** [comment]
Sorry I pushed some new commits before merging your MR, can you resolve the conflicts, thank you :+1:

File: DisplayCAL/worker.py [hunk 1]
```diff
@@ -5637,12 +5638,12 @@ def enumerate_displays_and_ports(
...
                         # Get monitor description using win32api
                         device = util_win.get_active_display_device(
-                            monitors[i][b"Device"]
+                            monitors[i]["Device"]
                         )
                         if device:
-                            desc.append(device.DeviceString.decode(fs_enc, "replace"))
+                            desc.append(device.DeviceString)
                         # Deal with HiDPI - update monitor rect
-                        m_left, m_top, m_right, m_bottom = monitors[i][b"Monitor"]
```
Comment (eoyilmaz) [inline DisplayCAL/worker.py:5645]: yeah this was obviously wrong

File: DisplayCAL/worker.py [hunk 2]
```diff
@@ -5637,12 +5638,12 @@ def enumerate_displays_and_ports(
                     if sys.platform == "win32" and i < len(monitors):
                         # Get monitor description using win32api
                         device = util_win.get_active_display_device(
-                            monitors[i][b"Device"]
```
Comment (eoyilmaz) [inline DisplayCAL/worker.py:5640]: yeah this was obviously wrong

File: DisplayCAL/wxwindows.py [hunk 3]
```diff
@@ -5236,19 +5236,18 @@ def fancytext_Renderer_getCurrentColor(self):
 fancytext.Renderer.getCurrentFont = fancytext_Renderer_getCurrentFont
 
 
-def fancytext_RenderToRenderer(str, renderer, enclose=True):
-    str = safe_str(str, "UTF-8")
```
Comment (eoyilmaz) [inline DisplayCAL/wxwindows.py:5240]: Let me fix it in this way, I replaced with ``str_`` but ``text`` seems better :+1:

Comment (p5k369) [reply DisplayCAL/wxwindows.py:5240]: thank you for doing the merge

**Pull request merged**